File: doWhile.cpp
#include <iostream>
using namespace std;

int main()
{
    int bil;
    int batas;
    int i=1;
    char n='T';

    
    do {
    
        cout<< "Masukkan Bilangan : "<<endl;
        cin>>bil;
        cout<<endl;
        
        if (bil%2==0)
        {
            cout<<"merupakan bilangan genap"<<endl;
        }
        else
        {
            cout<<"merupakan bilangan ganjil"<<endl;
        }
        
        cout<<"Masukkkan bilangan lagi? [Y/T] :"<<endl;
        cin>>n;
    } while (n=='Y');

    return 0;
}

File: doWhile.java
import java.util.Scanner;


class Main {
    public static void main(String[] args) {
        Scanner myObj = new Scanner(System.in);
        String y, sbatas;
        int x, batas;
        int i = 1;
        
        System.out.println("Masukkan batas perulangan : ");
        sbatas = myObj.nextLine();
        batas = Integer.parseInt(sbatas);
        
        do {
        System.out.println("\nMasukkan bilangan : ");
        y = myObj.nextLine();
        x = Integer.parseInt(y);
        
            if (x%2==0)
                System.out.println("Bilangan " + x + " merupakan bilangan genap\n");
            else
                System.out.println("Bilangan " + x + " merupakan bilangan ganjil\n");
        
        i++;

        } while (i<=batas);
    }
}